src/routes/file-centric-tsv/utils.ts

```diff
@@ -110,9 +110,14 @@ export const createEsDocumentStream = async function*<DocumentType>(configs: {
   }
 };
 
-export const writeTsvStreamToResponse = async <Document>(
+type WritableTarget = {
+  // this is basically a subset of Express Response type, but this is easier to mock for testing
+  write: (str: string) => any;
+};
+
+export const writeTsvStreamToWritableTarget = async <Document>(
   stream: AsyncGenerator<Document[], void, unknown>,
-  res: Response,
+  res: WritableTarget,
```

Comment:
just a suggestion, maybe this should be called `writableTarget`

Reply:
👍